python-packages/order_utils/src/zero_ex/order_utils/__init__.py

```diff
@@ -10,19 +10,22 @@
 fence smart topic"``.
 """
 
+from enum import auto, Enum
 import json
-from typing import Dict
+from typing import Dict, Tuple
 from pkg_resources import resource_string
 
 from mypy_extensions import TypedDict
 
-from eth_utils import is_address, keccak, to_checksum_address, to_bytes
+from eth_utils import is_address, keccak, to_bytes, to_checksum_address
 from web3 import Web3
-from web3.utils import datatypes
 import web3.exceptions
+from web3.utils import datatypes
 
+from zero_ex.dev_utils.type_assertions import assert_is_hex_string
 
-class Constants:  # pylint: disable=too-few-public-methods
+
+class _Constants:
```

Comment:
It implies that it's "private/internal."  More concretely, though, it means that the automatic documentation generation will not include this class in the doc page.

Reply:
I went with a class because member variables that are initialized in the declaration are considered "static", meaning that they're only initialized once.  (So, for us, the json artifacts are only read from disk once, at declaration time.)  I went back and forth on whether to call the class `_StaticData` instead.  Thinking more, maybe it's appropriate to have a `class Constants` (no underscore, so publicly documented) with things like `contract_name_to_abi` and `null_address`, things that other people might want to use; and to have a separate `class _StaticData` (so "private," not documented) that contains things that are only used internally, like precomputed keccak hashes and stuff.  Just brainstorming...  Dunno how much people would re-use that stuff right now, and in the future we'll probably have separate public packages to more properly expose things like ABI's and contract addresses.